source/server/BUILD

```diff
@@ -465,6 +464,7 @@ envoy_cc_library(
         "//source/common/upstream:cluster_manager_lib",
         "//source/common/upstream:health_discovery_service_lib",
         "//source/common/version:version_lib",
+        "//source/extensions/transport_sockets/tls:config",
```

Comment:
Can this depend on `//source/common/tls:context_lib` instead?

Reply:
I think not without TLS being a required extension.  Have to do one or the other.  have a preference?

Reply:
I think it's cleaner to have the dependency her on source/common if possible. It's probably fine to make tls a required extension.